org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ParameterizedExpressionsTestLanguageGrammarAccess.java

```diff
@@ -174,14 +178,14 @@
 		private final Assignment cBodyAssignment_1_0 = (Assignment)cGroup_1.eContents().get(0);
 		private final RuleCall cBodyBlockParserRuleCall_1_0_0 = (RuleCall)cBodyAssignment_1_0.eContents().get(0);
 		
-		//fragment FunctionBody <Yield, Expression> *:
-		//	<Expression> body=Block<Yield> | <!Expression> body=Block<Yield>?;
+		//fragment FunctionBody <Yield,Expression>*:
+		//	< Expression > body=Block<Yield> | < ! Expression > body=Block<Yield>?;
```

Comment:
@ArneDeutsch Unexpected spaces around the `!` and for the angular brackets. `<` and `>`. Please also check the formatting rules for the boolean operators in argument lists. They should be surrounded by a space ` & ` and ` | `

Reply:
Should be fixed with https://github.com/eclipse/xtext-core/issues/1489